modules/sfp_ivre.py

```diff
@@ -0,0 +1,342 @@
+# -*- coding: utf-8 -*-
+# -------------------------------------------------------------------------------
+# Name:         sfp_ivre
+# Purpose:      Query an IVRE instance
+#
+# Author:      Pierre Lalet <pierre@droids-corp.org>
+#
+# Created:     2021-09-14
+# Copyright:   (c) Pierre Lalet
+# Licence:     GPL
+# -------------------------------------------------------------------------------
+
+import hashlib
+
+from netaddr import IPNetwork
+from ivre.db import db, DBPassive
```

Comment:
Any way this could be done using pure REST instead of relying on another library?